webapp/components/User/User.vue

```diff
@@ -1,32 +1,37 @@
 <template>
-  <div class="user" v-if="displayAnonymous">
-    <hc-avatar v-if="showAvatar" class="avatar" />
-    <div>
-      <b class="username">{{ $t('profile.userAnonym') }}</b>
-    </div>
+  <div class="user-teaser" v-if="displayAnonymous">
+    <user-avatar v-if="showAvatar" />
+    <span class="info anonymous">{{ $t('profile.userAnonym') }}</span>
   </div>
-  <dropdown v-else :class="{ 'disabled-content': user.disabled }" placement="top-start" offset="0">
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-08T15:44:02Z" title="Wednesday, January 8th 2020, 4:44:02 pm +01:00">Jan 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -9,7 +9,7 @@
     <template slot="default" slot-scope="{ openMenu, closeMenu, isOpen }">
       <nuxt-link :to="userLink" :class="['user', isOpen && 'active']">
         <div @mouseover="showPopover ? openMenu(true) : () => {}" @mouseleave="closeMenu(true)">
```


This `<div>` isn't doing anything in the DOM, just adding an extra layer. I see that it is important for some of the functionality of the component but since this is its only job I would try to move these things up or down a level (i.e. to `nuxt-link`, `template`, `dropdown` or `user-avatar`) – if possible. I know, we're actually in the process of refactoring `user-avatar` but removing the extra layer makes positioning the `avatar` easier!